s3pypi/core.py

```diff
@@ -89,6 +90,18 @@ def parse_distribution(path: Path) -> Distribution:
     return Distribution(name, version, path)
 
 
+def parse_distributions(paths: List[Path]) -> List[Distribution]:
+    dists = []
+    for path in paths:
+        if path.is_file():
+            dists.append(parse_distribution(path))
+        if path.is_dir():
```

Comment:
Might as well make this `elif`.

Reply:
Yep, I considered this, and felt that the code was quicker to read this way. It does absolutely cost an extra path comparison though. I am happy to change it if you want.